mercury/views/views.py

```diff
@@ -9,3 +11,22 @@ def get(self, request, **kwargs):
 
 class AboutPageView(TemplateView):
     template_name = "about.html"
+
+
+class EventAccess(TemplateView):
+    def post(self, request, *args, **kwargs):
+        event_code = request.POST.get("eventcode")
+        event_access_object = EventCodeAccess.objects.latest("id")
```

Comment:
Does this code only check the "latest" event code in the table? What if an earlier event code needs to be re-enabled? Or, what if a future event code is put in the database but shouldn't be enabled yet? 

Have you considered doing a get/select using a filter to try and find a matching object that matches the event code, then checking if that match is enabled?

@MichaelLally  can you review the behavior?

Reply:
Well thought David, I considered that scenario where you can search for the event code provided by the user in the database but what if it’s a past event code that is not yet disabled by the admin. 
Here I am not sure of the requirements, so I am just picking the top row for testing  rest of the functionalities.
@MichaelLally please clarify on the desired behavior

Reply:
If it is a "past event code" that is not disabled by the admin, then it is enabled, and should be viable. It is up to the admin to enable and disable event codes properly.